functions/Copy-DbaDbViewData.ps1

```diff
@@ -0,0 +1,452 @@
+function Copy-DbaDbViewData {
+    <#
+    .SYNOPSIS
+        Copies data from a SQL Server view to a table.
+
+    .DESCRIPTION
+        Copies data from a SQL Server view to a table using SQL Bulk Copy.
+        The same can be achieved also doing
+        $sourcetable = Invoke-DbaQuery -SqlInstance instance1 ... -As DataTable
+        Write-DbaDbTableData -SqlInstance ... -InputObject $sourcetable
+        but it will force buffering the contents on the table in memory (high RAM usage for large tables).
+        With this function, a streaming copy will be done in the most speedy and least resource-intensive way.
```

Comment:
Please remove line breaks and the alternative code...

```
Copies data from a SQL Server view to a table using SQL Bulk Copy.
With this function, a streaming copy will be done in the most efficient and least resource-intensive way.
```

The code alternative is not required in the docs. If you think it needs to be you can add it to the notes section.

Reply:
I will remove and resubmit

Reply:
complete